src/config.c

```diff
@@ -3295,6 +3295,7 @@ standardConfig static_configs[] = {
     createIntConfig("rdma-port", NULL, MODIFIABLE_CONFIG, 0, 65535, server.rdma_ctx_config.port, 0, INTEGER_CONFIG, NULL, updateRdmaPort),
     createIntConfig("rdma-rx-size", NULL, IMMUTABLE_CONFIG, 64 * 1024, 16 * 1024 * 1024, server.rdma_ctx_config.rx_size, 1024 * 1024, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("rdma-completion-vector", NULL, IMMUTABLE_CONFIG, -1, 1024, server.rdma_ctx_config.completion_vector, -1, INTEGER_CONFIG, NULL, NULL),
+    createIntConfig("repl-max-reads-per-io-event", NULL, MODIFIABLE_CONFIG, 1, INT_MAX, server.repl_max_reads_per_io_event, 25, INTEGER_CONFIG, NULL, NULL),
```

Comment:
Suggestion is to just start with a hardcoded value of 25, then we can evaluate making it better by making the behavior more dynamic.

Options:

Reply:
lol, maybe I'm too passive in my wording. I try to avoid acting like it's an order. We nicely asked for it :)

Reply:
It needs to be clear what we decided. Otherwise we might come back and discuss it again and again. :)

Reply:
Sure sure, I'll try to do that moving forward.